keel-sql/src/main/resources/db/changelog/20200515-add-unique-id-to-event.yml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: add-unique-id-to-event
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: event
+        columns:
+        - name: uid
+          type: varchar(255)
+          constraints:
+            nullable: true
+    - modifySql:
+        dbms: mysql
+        append:
+          value: " first"
+- changeSet:
+    id: add-ref-to-events-with-no-uid
+    author: fletch
+    changes:
+    - sql:
+        sql: >
+          update event
+          set json = json_insert(json, '$.ref', json->'$.id')
+          where not(json_contains_path(json, 'one', '$.uid'))
```

Comment:
I don't understand what this is checking, especially given the second argument...

Reply:
Yeah, that param is weird. Docs are here: https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-contains-path

Reply:
…It's basically saying "where there's no `$.uid` in the JSON"

Reply:
Never mind. Man, that's a crappy syntax for that function.

Reply:
Sure is.